rdr_service/tools/tool_libs/questionnaire_response_duplicate_cleanup.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import sys
+import pytz
+
+from calendar import monthrange
+from datetime import datetime, timedelta
+from rdr_service.services.response_duplication_detector import ResponseDuplicationDetector
+
+_logger = logging.getLogger("rdr_logger")
+
+# Tool_cmd and tool_desc name are required.
+# Remember to add/update bash completion in 'tool_lib/tools.bash'
+tool_cmd = "qr_duplicate_fix"
+tool_desc = "Tool to run the questionnaire response duplicate detection logic over a long period of time."
+
+
+def run():
+
+    # Starting from year of oldest observed duplicates
+    start = datetime(2017, 1, 31, 23, 59, 59, 999999, pytz.UTC)
+    rdd = ResponseDuplicationDetector()
```

Comment:
Please always name your variable adequately for what they describe, maybe `response_detector` would be better?